lib/components/modal_progress_indicator.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:flutter/material.dart';
+
+class ModalProgressHUD extends StatelessWidget {
+  final bool inAsyncCall;
+  final double opacity;
+  final Color color;
+  final Offset offset;
+  final bool dismissible;
+  final Widget child;
+  final String indicatorStatus;
+
+  ModalProgressHUD({
+    Key key,
+    @required this.inAsyncCall,
+    @required this.indicatorStatus,
+    this.opacity = 0.3,
+    this.color = Colors.grey,
```

Comment:
make it black as it is changing the theme of the app

Reply:
> make it black as it is changing the theme of the app

This grey color dims the background when the spinner appears.
Spinner color has been changed.